rhobs/alerting/data_plane/prometheus.gitops_argocd_app_reconcile_alerts.yaml

```
@@ -17,6 +17,7 @@ spec:
           Argo CD in namespace {{ $labels.namespace }}
           has failed to reconcile at least 99% of applications
           within 4 seconds over the past 30 minutes
+        namespace: '{{ .Labels.dest_namespace }}'
```

I haven't tested this but shouldn't this use `$labels`?
```suggestion
        namespace: '{{ $labels.dest_namespace }}'
```

---

fixed